contributing.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
```

Comment:
I want to add something about how there is no advice that'll work for everyone, and all advice is subjective, and to avoid words like 'should', being aware of tone in how advice is given (suggestions, not sources of truth), and that it can be helpful to explicitly call out situations where different people will find different things helpful (to help the reader understand that it's expected that they pick and choose what works for them)

I can add this after though